### PR TITLE
[FIX] crm: replace old lead scoring content with predictive lead scoring

### PR DESCRIPTION
**Pull request by meng-odoo**
Replaced the entire doc with updated information on predictive lead scoring and assigning leads to sales teams/salespeople based on predictive lead scoring.
Closes tasks #[2199596](https://www.odoo.com/web#id=2199596&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form), #[3103697](https://www.odoo.com/web#id=3103697&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form), #[2797535](https://www.odoo.com/web#id=2797535&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

**Comment by meng-odoo** [comment]
Hello @ksc-odoo and @hojo-odoo! This is a priority PR (based on a task from a PO), so if one of you could please give this a content review soon, that would be great :)

Hi @samueljlieber, tagging you at the same time for technical review since this is a priority. Please also forward-port to 16.0 when we're ready.

Thank you!

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (samueljlieber) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:5]: Lets italicize the whole _Odoo CRM_

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @meng-odoo, I've finished by technical review of this PR. Everything looks just about good to me, nice job :)

The one change I have is italicizing "_Odoo CRM_" instead of just "Odoo _CRM_" throughout the doc. If you approve of this change I will squash the commits :)

**Comment by meng-odoo** [comment]
Thank you @samueljlieber! Let's wait for a content review before merging, since this doc isn't *too* pressing :)

Also, @StraubCreative has told me to italicize only the [App Name] part of "Odoo [App Name]", but tagging him here in case it's up for discussion, since I agree that italicizing the whole thing makes sense when referring to the product as "Odoo [App Name]".

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (StraubCreative) [reply content/applications/sales/crm/track_leads/lead_scoring.rst]: She had it right the first time (asked ahead of time.)
I know it's confusing 😅 

Italicize just the app name please, as that's the product name.
When we're referring to the app itself, we italicize it. When we're generally talking about the noun, it does not get italicized.

Difference:
Odoo _Helpdesk_ vs. help desk
Odoo _Inventory_ vs. "...inventory valuation"

In the case of CRM, it's a little strange. You have to decide based on context (are we talking about the Odoo product, a navigational step, etc.).

**Comment by StraubCreative** [comment]
> Thank you @samueljlieber! Let's wait for a content review before merging, since this doc isn't _too_ pressing :)
> 
> Also, @StraubCreative has told me to italicize only the [App Name] part of "Odoo [App Name]", but tagging him here in case it's up for discussion, since I agree that italicizing the whole thing makes sense when referring to the product as "Odoo [App Name]".

Comment above https://github.com/odoo/documentation/pull/3972#discussion_r1154784967

**Comment by ksc-odoo** [comment]
> Thank you @samueljlieber! Let's wait for a content review before merging, since this doc isn't _too_ pressing :)
> 
> Also, @StraubCreative has told me to italicize only the [App Name] part of "Odoo [App Name]", but tagging him here in case it's up for discussion, since I agree that italicizing the whole thing makes sense when referring to the product as "Odoo [App Name]".

FYI: I plan on giving this doc a review today - I'll be sure to tag the appropriate folks when it's ready to move on.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:8]: The Odoo *CRM*...

I believe is how ZST would this formatted. So, be sure to go back and adjust those accordingly.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:16]: ...Odoo *CRM*...

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:21]: ...score open leads/opportunities. 

As a company processes opportunities through the CRM pipeline, Odoo
collects data on which opportunities are won and lost. Predictive lead scoring uses this data to
predict the probability of winning each new lead or opportunity. 

The more opportunities that are sent through the CRM pipeline, the more data Odoo collects, resulting in more accurate probabilities.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:38]: The probability of success updates automatically, as the opportunity progresses through the CRM
pipeline. When an opportunity moves to the next stage, its probability of success automatically
increases, according to the predictive lead scoring algorithm.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:43]: ...Odoo *CRM*...

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:47]: *Quick Note: Since "Predictive Lead Scoring" is capitalized here, it might be worth considering making it capitalized throughout - for uniformity sake. What do you think?

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:51]: Then, click on the dropdown menu to choose which variables the predictive lead scoring feature will take into
account.

OR

Then, click on the dropdown menu to choose which variables the Predictive Lead Scoring feature will take into account.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:55]: ...Predictive Lead Scoring...

Again, another case for capitalization throughout...

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:78]: Change the probability (manually)
OR
Manually change the probability

(just suggestions, nothing required)

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:82]: An opportunity's probability of success can be changed manually on the opportunity form. Click on the probability number to edit it.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:87]: ...The probability will no longer update automatically, as the opportunity moves through each stage of the pipeline.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:97]: consider a brief intro sentence here to break things up a bit, and lead into the next portion of the doc. Just a thought...

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:101]: Odoo *CRM*...

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:102]: NOTE: I feel like if Rule-Based Assignment is capitalized in the Settings of the app, perhaps it should be capitalized everywhere else, as well? What do you think? It's the same "uniformity" suggestion from above.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:118]: for the "circular arrow icon" - do you want to consider using quotations around it, to make it stand out a bit more? Totally not a required change, but definitely something to consider.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:137]: To create an assignment rule, based on the probability of an opportunity's success, click on the...

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:141]: ...and select :guilabel:`Probability`.

From the middle dropdown menu, select the desired equation symbol — most likely the symbol for *greater than*,
*less than*, *greater than or equal to*, or *less than or equal to*. 

Then, in the far right space, enter the desired number value of the probability. Lastly, click :guilabel:`Save` to save changes.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:145]: To configure an assignment rule such that a sales team receives leads that have a 20% (or greater) probability of
success, create a :guilabel:`Domain` line that reads: `Probability >= 20`.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:149]: ...greater than (or equal to) twenty percent.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:152]: ...:guilabel:`Members` tab, then edit the...

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/track_leads/lead_scoring.rst:159]: If automatic lead assignment is configured in the settings, both the sales team and individual team
members have the option to :guilabel:`Skip auto assignment`. 

Check this box to omit a particular sales team (or salesperson) from automatically being assigned leads by Odoo's rule-based assignment feature. If :guilabel:`Skip auto assignment` is activated, the sales team (or salesperson) can still be assigned leads manually.

**Review by ksc-odoo — COMMENTED** [review]
Great work, @meng-odoo !!! Just finished my Peer Review and only had a few minor comments/suggestions - most of them dealing with capitalizing/uniformity for a couple of features, but other than that - I think it's ready for the next stage!

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (meng-odoo) [reply content/applications/sales/crm/track_leads/lead_scoring.rst:47]: I thought about doing this, however "predictive lead scoring" is a method used by other CRM companies as well, rather than a feature unique to (or invented by) Odoo, so I thought it best to keep it lowercase.

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (meng-odoo) [reply content/applications/sales/crm/track_leads/lead_scoring.rst:97]: Good idea, thanks :)

File: content/applications/sales/crm/track_leads/lead_scoring.rst
Comment (meng-odoo) [reply content/applications/sales/crm/track_leads/lead_scoring.rst:102]: Same kind of deal as "predictive lead scoring"; assigning leads based on rules is a common CRM feature, so I thought it best to keep this lowercase, even though the exact phrasing "rule-based assignment" is not necessarily commonly used.

**Comment by meng-odoo** [comment]
Thank you @ksc-odoo!!

@StraubCreative this is ready for final review :)

**Review by StraubCreative — APPROVED** [review]
Hi @meng-odoo 

Great PR. This was a pleasure to read (and I learned a bunch 😄)

I have a few suggestions below for your consideration which I think could improve a couple things, but either way we're g2g 🚀 

Delegating the merge command to you, pending your decision to incorporate last round of changes or merge immediately.

@robodoo delegate+

**Comment by meng-odoo** [comment]
Thanks @StraubCreative, I incorporated your suggestions in the latest commit. Merging now :)

@robodoo r+

**Pull request closed without merging**
